src/stories/VisCanvas.stories.tsx

```diff
@@ -60,7 +73,7 @@ export const GraphTitle = Template.bind({});
 GraphTitle.args = {
   abscissaConfig: { domain: [0, 3], showGrid: true, isIndexAxis: true },
   ordinateConfig: { domain: [50, 100], showGrid: true, isIndexAxis: true },
-  canvasTitle: 'This is a graph',
+  title: 'This is a graph',
 };
```

Comment:
Maybe also add a story for `fill`? (assuming `cover` becomes the new default)

Reply:
Well, `layout` is not supposed to be seen by `VisCanvas`...

Reply:
Ah of course, sorry 😄